.github/workflows/compatibility.yml

```diff
@@ -12,13 +12,13 @@ jobs:
       max-parallel: 10
       matrix:
         # Minimum support version, ..., Latest version, `beta`.
-        woocommerce: [ '6.2.2', '6.3.1', '6.4.1', '6.5.1', '6.6.1', '6.7.0', 'latest', 'beta' ]
+        woocommerce: [ '6.4.1', '6.5.1', '6.6.1', '6.7.0', 'latest', 'beta' ]
         wordpress:   [ 'latest' ]
         gutenberg:   [ 'latest' ]
         php:         [ '7.4' ]
         include:
           # Edge case: oldest dependencies compatibility
-          - woocommerce: '6.2.2'
+          - woocommerce: '6.4.1'
```

Comment:
This combination is failing in the PHP unit tests. I can't replicate locally. I get a fatal error when using Gutenberg 11.4.0.
For newer versions, I do get the `{"success":false,"data":"You aren\u2019t authorized to do that."}{"result":"failure"}` error but my test doesn't fail with `Failed asserting that an array is empty.`.

@brianyu28  I'm thinking of updating the workflows in a separate PR so we can move forward and prepare the package for testing while sorting things out. Do you think that'd be reasonable?

Reply:
Yeah, I agree it makes sense to get the testing package out first. The CI test updates I believe are meant to happen during the first week of the release cycle but doesn't need to be on Monday.

Reply:
Cool. Reverting the commit that updates the workflows for now. Thanks!